app/layout.tsx

```diff
@@ -18,19 +19,25 @@ export interface Metadata {
 }
 
 export const defaultMetadata: Metadata = {
-  title: 'Keploy | Open Source Stubs and API Test Generator for Developer',
-  keywords: "Integration testing, e2e Testing, ai testing, Unit Testing, API Testing, open source ai testing tool, Service Mocking, Dependency Mocking",
-  description: 'Keploy is AI based test case and stubs/mocks generator for integration and unit testing. 90% test coverage in minutes with open source testing tool',
+  title: "Keploy | Open Source Stubs and API Test Generator for Developer",
+  keywords:
+    "Integration testing, e2e Testing, ai testing, Unit Testing, API Testing, open source ai testing tool, Service Mocking, Dependency Mocking",
+  description:
+    "Keploy is AI based test case and stubs/mocks generator for integration and unit testing. 90% test coverage in minutes with open source testing tool",
```

Comment:
why is this changed?

Reply:
Hi @shivamsouravjha , nothing changed here, thats just auto formatting